src/cli/__tests__/getConfiguration.test.js

```diff
@@ -36,21 +30,6 @@ test('with options from arguments and prompt merge', async () => {
   );
 });
 
-test('without stable version available', async () => {
```

Comment:
Why did we remove this test?

Reply:
it's moved to postProcess as it's only used there